go/client/chat_svc_handler.go

```diff
@@ -152,6 +152,17 @@ func (c *chatServiceHandler) ReadV1(ctx context.Context, opts readOptionsV1) Rep
 			continue
 		}
 
+		selfUID := c.G().Env.GetUID()
+		if selfUID.IsNil() {
+			c.G().Log.Error("Could not get self UID for api")
```

Comment:
Maybe make this debug? This would get sent to the user since it is `Error`, and it will just look confusing.